.devcontainer/Dockerfile

```diff
@@ -30,27 +30,27 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update -q && \
 RUN source "/opt/ros/${ROS_DISTRO}/setup.bash"
 
 # Install bosdyn_msgs package
```

Comment:
```suggestion
ARG SPOT_SDK_VERSION="4.1.0"

# Install bosdyn_msgs package
```

Reply:
Done in https://github.com/bdaiinstitute/spot_ros2/pull/495/commits/88f0b6d9cd1abeffe4d8d6f5d87c1f926ed25684.